### PR TITLE
feat: add option for docker image rebuild

### LINKED ISSUE #715: Have to delete action container in order to run new changes
I'm unable to find any flag or mention of making act rebuild the action image. I am forced to delete the image which is very time consuming and misses out on layer caching.

Is there a way to tell act to run my action and build the image every time ?

### PR DESCRIPTION
**Pull request by catthehacker**
Adds option to rebuild local action docker images
which is disabled by default (old behaviour)

Fixes #715

Signed-off-by: hackercat <me@hackerc.at>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,7 +184,9 @@ It will save that information to `~/.actrc`, please refer to [Configuration](#co
       --privileged                       use privileged mode
   -p, --pull                             pull docker image(s) even if already present
   -q, --quiet                            disable logging of output from steps
+      --rebuild                          rebuild local action docker image(s) even if already present (default true)
   -r, --reuse                            reuse action containers to maintain state
+      --rm                               automatically remove containers just before exit
```
Comment (cplee) [inline README.md:189]: Is this implemented?

File: pkg/runner/runner_test.go [hunk 2]
```diff
@@ -57,6 +57,7 @@ func runTestJobFile(ctx context.Context, t *testing.T, tjfi TestJobFileInfo) {
 		assert.Nil(t, err, workdir)
 		fullWorkflowPath := filepath.Join(workdir, tjfi.workflowPath)
 		runnerConfig := &Config{
+			ForceRebuild:          true,
```
Comment (cplee) [inline pkg/runner/runner_test.go:60]: This should be default false to preserve current behavior.

File: README.md (hunk 1 above)
Comment (catthehacker) [reply README.md:189]: This was implemented in #714 but missed on adding to readme
 `--container-cap-add` missed in #716

File: pkg/runner/runner_test.go (hunk 2 above)
Comment (catthehacker) [reply pkg/runner/runner_test.go:60]: Will correct that

File: cmd/root.go [hunk 3]
```diff
@@ -45,6 +45,7 @@ func Execute(ctx context.Context, version string) {
 	rootCmd.Flags().BoolVarP(&input.reuseContainers, "reuse", "r", false, "reuse action containers to maintain state")
 	rootCmd.Flags().BoolVarP(&input.bindWorkdir, "bind", "b", false, "bind working directory to container, rather than copy")
 	rootCmd.Flags().BoolVarP(&input.forcePull, "pull", "p", false, "pull docker image(s) even if already present")
+	rootCmd.Flags().BoolVarP(&input.forceRebuild, "rebuild", "", true, "rebuild local action docker image(s) even if already present")
```
Comment (cplee) [inline cmd/root.go:48]: Can we stick with default of false?

**Review by cplee — CHANGES_REQUESTED** [review]

**Review by cplee — APPROVED** [review]

**Pull request merged**